examples/js/exporters/GLTFExporter.js

```diff
@@ -58,19 +58,22 @@ THREE.GLTFExporter.prototype = {
 	constructor: THREE.GLTFExporter,
 
 	/**
-	 * Parse scenes and generate GLTF output
+	 * Parse scenes and generate an object containing the glTF JSON, bin blob, and image urls/file names.
 	 * @param  {THREE.Scene or [THREE.Scenes]} input   THREE.Scene or Array of THREE.Scenes
 	 * @param  {Function} onDone  Callback on completed
+	 * @param  {Function} onError Callback on error
 	 * @param  {Object} options options
 	 */
-	parse: function ( input, onDone, options ) {
+	parseChunks: function ( input, onDone, onError, options ) {
 
 		var DEFAULT_OPTIONS = {
 			binary: false,
-			trs: false,
+			binFileName: "scene.bin",
+			trs: true,
 			onlyVisible: true,
 			truncateDrawRange: true,
-			embedImages: true,
+			embedImages: false,
+			useImageFileNames: true,
```

Comment:
What's the use case for returning names of files without their data? And what happens if a texture references a canvas element?

Reply:
See my comment above.